integration-tests/src/main/java/org/apache/druid/testing/utils/DruidClusterAdminClient.java

```diff
@@ -185,7 +187,23 @@ private void waitUntilInstanceReady(final String host)
             return response.getStatus().equals(HttpResponseStatus.OK);
           }
           catch (Throwable e) {
-            LOG.error(e, "");
+            //
+            // supress logging for some specific exceptions to reduce excessive stack trace messages when waiting druid nodes to start up
+            //
+            if (e.getCause() instanceof ChannelException) {
+              Throwable channelException = e.getCause();
+
+              if (channelException.getCause() instanceof ClosedChannelException) {
+                LOG.error("Channel Closed");
+              } else if (!"Channel disconnected".equals(channelException.getMessage())) {
+                // for 'Channel Disconnected', there's no need to log the error message
+                // because the underlying http client has already log this kind of message
+                LOG.error(e, "");
+              }
```

Comment:
I think this is easier to read, and a single line log message shouldn't add too much noise to the logs, but it is still a marker that says the channel disconnected

```suggestion
              } else if ("Channel disconnected".equals(channelException.getMessage())) {
                LOG.error("Channel disconnected: " + e.getMessage());
              } else {
                  LOG.error(e, "");
              }
```

Reply:
In this case, `e.getMessages` returns "Channel disconnected", so the 
```
LOG.error("Channel disconnected: " + e.getMessage());
```
outputs a text as 'Channel disconnected: Channel disconnected'.

I would change this line to ```LOG.error(e.getMessage());```